.github/workflows/build.yaml

```
@@ -0,0 +1,82 @@
+---
+name: build
+on:
+  push:
+    branches:
+      - main
+    tags:
+      - v*.*.*
+  workflow_dispatch:
+
+jobs:
+  build-java-source:
+    name: build jar file
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v3
+
+      - name: Set up Java build environmant
+        uses: actions/setup-java@v2
+        with:
+          distribution: 'temurin'
+          java-version: '17'
+          cache: 'maven'
+
+      - name: Build with Maven
+        run: mvn package -DskipTests
```

We shouldn't skip the tests on this repository.
```suggestion
        run: mvn package
```